how-to-docs/drone-how-to.md

```diff
@@ -187,12 +187,12 @@ Images hosted on [Artifactory](https://docker.digital.homeoffice.gov.uk) are pri
 
 If your repository is hosted publicly on GitHub, you shouldn't publish your images to Artifactory. Artifactory is only used to publish private images. [You should use Quay to publish your public images](#publishing-to-quay).
 
-[Raise a ticket for a new Artifactory robot](https://github.com/UKHomeOffice/application-container-platform-bau/). You have to pick a name for it.  You should be supplied a robot token in response.
+[Submit a support request for a new Artifactory robot](https://hub.acp.homeoffice.gov.uk/help/support/requests/new/artifactory-bot). You should be supplied a robot token in response.
 
-You can inject the robot's token supplied to you in your raised ticket to the Platform team with:
+You can inject the robot's token that has been supplied to you with:
 
 ```
-$ drone secret add --image="<your_image>" UKHomeOffice/<your_repo> DOCKER_ARTIFACTORY_PASSWORD your_robot_token
+$ drone secret add --image="<image_name>" ukhomeoffice/<your_gitlab_repo> DOCKER_ARTIFACTORY_PASSWORD your_robot_token
```

Comment:
I can't recall if the new Drone allows you to do this, or if you need to set up the secret via the web UI – it's worth checking/trying with a dummy repository.

Reply:
Adam was able to add a secret this way with new Drone so it does work.

Reply:
Perfect 👍